ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -314,7 +315,20 @@ const GENE_DISEASE_DETAIL_SECTIONS = [
   },
 ]
 
-const RNA_SEQ_DETAIL_FIELDS = ['zScore', 'pValue', 'pAdjust']
+const RNA_SEQ_COLUMNS = [
```

Comment:
It would be much cleaner to use `map` to set the values for this rather than do all this copy-paste
```
const RNA_SEQ_COLUMNS = [
  { name: 'individual', content: '', width: 3 },
  ...['zScore', 'pValue', 'pAdjust'].map(name => (
    { name, content: camelcaseToTitlecase(name).replace(' ', '-'), width: 3, format: row => row[name].toPrecision(3) }
  ),
]
```